src/components/reservation.jsx

```diff
@@ -0,0 +1,141 @@
+import { useDispatch, useSelector } from "react-redux";
+import { useEffect } from "react";
+import Button from "react-bootstrap/Button";
+import Card from "react-bootstrap/Card";
+import "../styles/reservation.css";
+import { fetchReservations, fetchReservationDetails } from "../redux/reservationsSlice";
+import { deleteReservationApi } from "../redux/reservationDeleteSlice";
+import Swal from "sweetalert2";
+import { Navigation, Pagination, Scrollbar, A11y } from 'swiper/modules';
+
+import { Swiper, SwiperSlide } from "swiper/react";
+import "swiper/css";
+import "swiper/css/navigation";
+import "swiper/css/pagination";
+import "swiper/css/scrollbar";
+import "../styles/Home.css";
+
+const Reservation = () => {
```

Comment:
- [ ] When I create multiple users, it seems everybody is sharing the same reservations, including the new ones I create:

User 1:

![image](https://github.com/danielamoreno699/wanderpal-frontend/assets/57421823/93a3680c-d3bd-4d1f-aa6a-14483056df1e)

User 2:

![image](https://github.com/danielamoreno699/wanderpal-frontend/assets/57421823/1cdc5e96-0262-4caf-a52f-42518ac822f8)

Kindly, make it so each user has their own reservations.

Tip: The backend interface for this has already been done.  

That way, your app will be more useful for making reservations. 👍